app/v1/otherReliefs/amend/model/response/AmendOtherReliefsResponse.scala

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v1.otherReliefs.amend.model.response
+
+import api.hateoas.{HateoasData, HateoasLinks, HateoasLinksFactory, Link}
+import config.AppConfig
+
+object AmendOtherReliefsResponse extends HateoasLinks {
```

Comment:
There is a copy of this in `v1.models.response.amendOtherReliefs` that should be deleted in favour of this one. (Doesn't look like this one is being used.)